### DISCUSSION TITLE
python-pywal and leftwm

### DISCUSSION THREAD

**Discussion by arcolinuxz** [leftwm/leftwm#872] [General]
Hi guys,

I opened up a first issue for the migration from toml to ron.

https://github.com/dylanaraps/pywal/issues/688

Please take over the post or correct it in any way.

We use python-pywal to make leftwm even nicer in design.

It gives us a toml file to copy/paste but not a ron file obviously.

see and install https://github.com/arcolinux/arcolinux-leftwm-theme-db-nemesis to see it in practice

All on ArcoLinux so Arch based - sudo pacman -S python-pywal

Maybe add some more info to help Dylan.

**Comment by VuiMuich** [comment]
Would this work as template?
```
(
border_width: 2,
margin: [4, 4],
default_border_color: "{background}",
floating_border_color: "{color6}",
focused_border_color: "{foreground}",
)
```

Tested this now, and works fine.
I put it in `~/.config/wal/templates/colors-leftwm-theme.ron` and used this to apply the theme:
```sh
wal -i <path-to>/background.png
leftwm command "LoadTheme /home/username/.cache/wal/colors-leftwm-theme.ron
```

**Reply by VuiMuich** [reply]
I played a bit with `pywal` and to me it seems like the best solution would be to strip down the template to only the three lines with the color fields and `sed` them into the `theme.ron` of the theme before loading the theme.

**Reply by erikdubois** [reply]
you can suggest this on the pywal website

**Reply by VuiMuich** [reply]
> you can suggest this on the pywal website

I will prepare a suggestion for their wiki.
